.github/workflows/main.yml

```diff
@@ -13,6 +13,10 @@ on:
       - '**.md'
   workflow_dispatch: []
 
+concurrency:
```

Comment:
Can you please add some comments here that paraphrase the text you put in the PR description.

Reply:
Done in https://github.com/oracle/graal/pull/3431/commits/f133a6407f422214ce7c797a74695d21feff9533